File: week_5/day_1/warmup/switchup.rb
require "pry"

def switch_up str   
    str_switched_up = []
    str.chars.each do |letter|
        if letter == letter.upcase
            str_switched_up.push letter.downcase
        else 
            str_switched_up.push letter.upcase
        end
    end
    str_switched_up.join
end

binding.pry 

File: week_5/day_1/warmup/switchUp.js
function switchUp(str) {
    var str_switched = [];
    for (var letter = 0; letter < str.length; letter++) {
        if (str[letter] === str[letter].toUpperCase()) {
            str_switched.push(str[letter].toLowerCase());
        } else {
            str_switched.push(str[letter].toUpperCase());
        }
    }
    return str_switched.join('')
}